src/pages/my-buses/my-buses.html

```diff
@@ -1,6 +1,6 @@
 <ion-header>
   <ion-navbar color="primary">
-    <button ion-button menuToggle>
+    <button ion-button menuToggle aria-label="opens the menu" tabindex=1>
```

Comment:
Probably outside of the scope of this PR, but, ~~is there a code-reuse way to not have the same menu in four pages?~~

yeesh, my quick Google search reveals that the answer is "maybe, but it's super complicated" 😨